scripts/management/commands/secret_santa.py

```diff
@@ -24,31 +24,41 @@ def handle(self, *args, **options):
         for count, user in enumerate(users):
             to_from_dict['Par %s' % (count+1)] = {'To': user.user.get_full_name(),
                                                   'From': users[random_users[count]].user.get_full_name()}
-
             self.send_mail(user, users[random_users[count]])
 
         self.write_to_file(to_from_dict)
-        self.mail_to_committee(to_from_dict)
+        self.mail_to_committee()
 
     # Function for generating a list with random numbers corresponding to
     # the indexes of the list of users attending the event.
     @staticmethod
     def generate_random_list(users):
         to_from = random.sample(range(len(users)), len(users))
-        while(not Command.check_random_list(to_from)):
-            to_from = random.sample(range(len(users)), len(users))
+        Command.check_random_list(to_from)
         return to_from
 
     # Function for checking that the random list of indexes is valid.
     # Checks the index of the original list of users against the randomly
     # generated number. If index and random number is the same, then someone
     # got themselves as secret Santa.
     @staticmethod
-    def check_random_list(random):
-        for user_index, user2_index in enumerate(random):
-            if (user_index == user2_index):
-                return False
-        return True
+    def check_random_list(random_list):
```

Comment:
If this function makes sure the map follows the set constraints rather than just checking that it doesn't violate the constraints, the name should probably be changed from `check` to `fix` or something along those lines.